PR: Update Changelog, Announcements and metadata files for 6.0.5 (#24051)
spyder-ide/spyder: changelogs/Spyder-6.md

----

## Version 6.0.5 (2025/03/26)

### New features

> * Option added to disable project files searches in the File Switcher.
> * Support displaying environments with the same name over the IPython Console `New console in environment` menu.

Reviewer: ```suggestion
* Add option to the Projects options menu to disable file searches in the Switcher.
* Support displaying environments with the same name in the IPython Console `New console in environment` menu.
```

### Important fixes

> * Fix `Check for updates at startup` option value update when an update is declined.

Reviewer: ```suggestion
* Fix `Check for updates at startup` option when an update is declined.
```

* Remove `mamba` from Spyder installers.
* Several improvements to the Variable Explorer messages shown when a variable can't be displayed.

> * Fix external kernel connection dialog to prevent trying to connect when the connection file doesn't exists.
> * Several fixes related with the Run plugin and working directory options usage to run and debug files.
> * Fix conda executable validation when creating kernels and improve feedback in case it was not found.
> * Add message related with Pixi created environments support when starting kernels.
> * Improve message related with Spyder icon theme/fonts load not being possible.

Reviewer: ```suggestion
* Prevent error in `Connect to an existing kernel` dialog when the connection file doesn't exist.
* Several fixes related to the Run plugin and working directory options used to run and debug files.
* Fix conda executable validation when creating kernels and improve feedback in case it's not found.
* Add message related to support for Pixi environments when starting kernels.
* Improve message related to loading the Spyder icon theme not being possible.
```

* Prevent Spyder softlock when lossing focus while the tour is being shown.
* Fixes to better handle errors when trying to load `.spydata` files in the Variable Explorer.

> * Fix Editor code folding and indent guides for new editors.

Reviewer: ```suggestion
* Fix Editor code folding and indent guides for cloned editors.
```
